tensorflow/compiler/xla/service/gpu/ir_emitter.h

```diff
@@ -177,9 +184,13 @@ class IrEmitter : public DfsHloVisitorWithDefault,
   // management rules, their memory is owned by the module.
   llvm::IRBuilder<> b_;
 
+  // Wrapper class providing access to builder and LLVMTargetFeatures;
```

Comment:
Please end sentences with a period (although this isn't really a sentence, so no punctuation would be OK too).